web/Views/Shared/_Layout.cshtml

```diff
@@ -1,102 +1,104 @@
-            <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index">BricksAndHearts</a>
+            <a class="navbar-brand fw-bold ms-4 me-5" asp-controller="Home" asp-action="Index">Bricks & Hearts</a>
+
             <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target=".navbar-collapse" aria-controls="navbarSupportedContent"
                     aria-expanded="false" aria-label="Toggle navigation">
                 <span class="navbar-toggler-icon"></span>
             </button>
-            <div class="navbar-collapse collapse d-sm-inline-flex justify-content-between">
-                <ul class="navbar-nav flex-grow-1">
-                    <li class="nav-item">
-                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
-                    </li>
-                    @if (User.IsInRole("Admin"))
-                    {
-                        <li class="nav-item dropdown">
-                            <a class="nav-link active dropdown-toggle" href="#" id="navbarDropdownMenuLink" data-bs-toggle="dropdown">
-                                Admins
-                            </a>
-                            <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-                                <a class="dropdown-item text-dark" asp-area="" asp-controller="Admin" asp-action="GetAdminList">Admin List</a>
-                                <a class="dropdown-item text-dark" asp-area="" asp-controller="Admin" asp-action="LandlordList">Landlord List</a>
-                            </div>
-                        </li>
-                    }
-                    <li class="nav-item">
-                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a>
-                    </li>
-                    @if (User.IsInRole("Landlord"))
-                    {
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="MyProfile">Profile</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Landlord" asp-action="ViewProperties">View Properties</a>
-                        </li>
-                    }
-                </ul>
 
-                <ul class="navbar-nav ms-auto">
+            <div class="collapse navbar-collapse">
+                <div class="navbar-nav me-auto">
+                    @* Change this link to landlord index when created *@
+                    <a class="nav-link text-dark mx-3" asp-controller="Landlord" asp-action="RegisterGet">List your properties</a>

```

Comment:
Can you do this and just send them to the profile for now?

Reply:
Do you mean to link to the profile instead of the registration form? (The form redirects to the profile if you're a landlord already). Or to add an "index" action that redirects to the profile?

Reply:
Link to the profile instead of the registration

Reply:
Done!

Reply:
Ah sorry I meant if they're registered, link straight to profile, if not link to register. Trying to rush through reviews 😅 

I know register will redirect, but it feels wrong to be relying on that. Maybe that's silly though. Definitely don't want to link straight to profile if they're not registered as it will just error right?

Reply:
Yeah profile gives a 403 if they're not registered. I've added your suggestion!